### ISSUE TITLE
Setup Gulp to Optimize Workflow

### ISSUE THREAD

**Issue by Jov03** [UdacityFrontEndScholarship/udabuddy#68]
I would like to setup gulp on this project in order to turn on live editing and sass preprocessing on this project. Can I make the PR after setting it up.

**Comment by littlestar642** [comment]
@Jov03 Earlier when we were deciding the stack for the project, i gave a thought over to automatize the workflow using some taskmanager,particularly grunt,but then it occured to me that people here are facing problem with git push and pull only and other basic stuff and we cant really add a tech that they are completely new to. So i left that for the time. What do you think,should we integrate gulp here?

**Comment by Jov03** [comment]
@littlestar642 i just integrated browser sync i haven't introduced sass or anything.Just browser sync so that we can preview changes quickly without refreshing the page over and over again.

**Comment by karkranikhil** [comment]
We can automate the workflow with any of the tech grunt or gulp. My personal preference is Gulp @littlestar642

**Comment by Jov03** [comment]
Yeah because gulp uses streams allowing us to run parallel task and its code is in JavaScript 😄
